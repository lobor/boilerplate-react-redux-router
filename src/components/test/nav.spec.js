import React from 'react'
import styled from 'styled-components'
import renderer from 'react-test-renderer'
import Nav from './../nav'

describe('Test component Nav', () => {
	test('simple nav', () => {
	  const tree = renderer.create(<Nav />).toJSON()
	  expect(tree).toMatchSnapshot()
	})

	test('horizontal nav', () => {
	  const tree = renderer.create(<Nav horizontal />).toJSON()
	  expect(tree).toMatchSnapshot()
	})

	test('top nav', () => {
	  const tree = renderer.create(<Nav top />).toJSON()
	  expect(tree).toMatchSnapshot()
	})
});
