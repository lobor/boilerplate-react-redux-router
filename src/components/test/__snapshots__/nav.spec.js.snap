// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test component Nav horizontal nav 1`] = `
<nav
  className="sc-bdVaJa inDkyG"
/>
`;

exports[`Test component Nav simple nav 1`] = `
<nav
  className="sc-bdVaJa dIMaYf"
/>
`;

exports[`Test component Nav top nav 1`] = `
<nav
  className="sc-bdVaJa eJjEcC"
/>
`;
